### PR TITLE
fix: Normalize debug ids for minidumps

### PR DESCRIPTION
**Pull request by untitaker**
This leaves the behavior for incoming requests untouched, where the debug ids will be echoed back unmodified.

### CODE REVIEW

**Comment by untitaker** [comment]
@jan-auer pls check again

File: src/types.rs [hunk 1]
```diff
@@ -499,7 +501,19 @@ pub struct CompleteObjectInfo {
...
-    fn from(raw: RawObjectInfo) -> Self {
+    fn from(mut raw: RawObjectInfo) -> Self {
+        if let Some(ref raw_id) = raw.debug_id {
+            if let Ok(id) = DebugId::from_breakpad(raw_id.as_str()) {
+                raw.debug_id = Some(id.to_string());
+            }
+        }
+
+        if let Some(ref raw_id) = raw.code_id {
+            if let Ok(id) = raw_id.parse() {
```
Comment (jan-auer) [inline src/types.rs]: This parses `raw_id` into a String, which is a no-op. Run this instead:

```suggestion
            if let Ok(id) = raw_id.parse::<CodeId>() {
```

Comment (untitaker) [reply src/types.rs]: Good catch...

**Comment by untitaker** [comment]
Before we merge this we need to update snapshots in Sentry again or find an alternative solution there.

**Pull request merged**